.travis.yml

```diff
@@ -12,6 +12,7 @@ addons:
   apt:
     packages:
       - expect-dev # provides unbuffer utility
+      - libmagic1 # Required by module storage_import_image
```

Comment:
I think this should be added to .copier-answers.yml

Reply:
not sure how to add this dependency there :face_with_head_bandage:

Reply:
the issue is that we'll lose the dep while sync'ing w/ https://github.com/OCA/oca-addons-repo-template.
I think it's just fine to add it here https://github.com/OCA/storage/blob/14.0/.copier-answers.yml#L12
@Yajo any recommendation in such cases?

Reply:
Copier update will respect your git history, so as long as the answers-vs-code relation don't change, you won't get conflicts. However indeed it's better IMHO to just do `copier update` and add that new dependency.